.github/workflows/SDK-Suppressions-Label.yaml

```diff
@@ -0,0 +1,114 @@
+name: SDK Suppression Label
+
+on:
+  pull_request:
+    branches:
+      - main
+      - RPSaaSMaster
+      - release*
+    types:
+      - opened
+      - synchronize
+      - reopened
+
+  workflow_dispatch:
```

Comment:
Since your action needs to compare two commits, it doesn't make sense to trigger on `workflow_dispatch` with no inputs.  If you wanted this feature, you'd need to specify inputs like the two commit SHAs to compare.  Or remove the `workflow_dispatch` trigger.

Reply:
removed